packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -87,6 +97,92 @@ export default class PciProjectNewPaymentCtrl {
       });
   }
 
+  displayCucCloudMessage(type, suffix) {
+    this.CucCloudMessage[type](
+      this.$translate.instant(
+        `pci_project_new_payment_check_anti_fraud_case_${suffix}`,
+      ),
+      'pci.projects.new.payment',
+    );
+  }
+
+  displayAntiFraudMessage(validatingStep, order) {
+    const {
+      FRAUD_DOCS_REQUESTED,
+      FRAUD_MANUAL_REVIEW,
+    } = this.meEnums.ORDER_FOLLOW_UP_HISTORY_STATUS;
+
+    (validatingStep?.history || []).forEach(({ label }) => {
+      if ([FRAUD_MANUAL_REVIEW, FRAUD_DOCS_REQUESTED].includes(label)) {
+        this.trackPage('antifraud-verification');
```

Comment:
you can use `atInternet` rename in route itself if you want to change page tracking name: 

https://github.com/ovh/manager/blob/8abdbb6da14debec1410351dd9ae99b1e5ae473b/packages/manager/modules/web-paas/src/details/service/service.routing.js#L35

Reply:
No i can't, i used in case where the antifraud case appear.
So it's not when the page are initialized.

Thanks.

Reply:
@mohammed-zahaf isn't the method `trackEvent` more accurate in this case ? As you are making an hit onDisplayAntiFraudMessage.

Reply:
No, the requested event from tracking reporter is page.